### PR TITLE
Create installation-tips.md

### PR DESCRIPTION
**Pull request by susanshi**
Wanted to start a document to help trouble shoot installation issues

### CODE REVIEW

File: docs/troubleshoot/installation-tips.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+You must provide a TLS certificate for Ratify to use or enable RATIFY_CERT_ROTATION. Enable feature flag `RATIFY_CERT_ROTATION` if you don't have a ready to use TLS certificate, Ratify will rotate certificates automatically when they are about to expire.
+
+```bash
+# download the notation verification certificate
+curl -sSLO https://raw.githubusercontent.com/deislabs/ratify/main/test/testdata/notation.crt
+helm install ratify \
+    ratify/ratify --atomic \
+    --namespace gatekeeper-system \
+    --set-file notationCert=./notation.crt \
```
Comment (akashsinghal) [inline docs/troubleshoot/installation-tips.md:23]: for this example, should we remove the `notationCert` flag?

**Review by binbin-li — APPROVED** [review]
changes lgtm, per Akash's comment, I'm Ok leaving it ore removing it.

**Comment by susanshi** [comment]
Updated to "your notation cert" please review again, thanks @binbin-li @akashsinghal

**Review by binbin-li — APPROVED** [review]
lgtm

**Review by akashsinghal — APPROVED** [review]
thanks for the changes. lgtm

**Pull request merged**